lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -192,13 +189,17 @@ def add_invoice(post, money, options)
 
       def add_card_or_token(post, payment, options)
         payment = payment.split('|')[0] if payment.is_a?(String)
-        post[:payment][:payment_type_code] = 'creditcard'
+        add_payment_type(post[:payment])
         post[:payment][:creditcard] = payment_details(payment)
         post[:payment][:creditcard][:soft_descriptor] = options[:soft_descriptor] if options[:soft_descriptor]
       end
 
-      def add_payment_details(post, payment)
+      def add_payment_type(post)
         post[:payment_type_code] = 'creditcard'
```

Comment:
If the `payment_type` is only ever `creditcard` should we add this to `add_payment_details`?

Reply:
True, will do that!

Reply:
Actually there's a reason I did that because because `Verify` uses `post[:card]` and the other methods use `post[:creditcard]` which is why I move `payment_type` out. I'm actually going to update that and remove `add_payment_details` so it makes a bit more sense.